### PR TITLE
Remove legacy AFR endpoints

### PR DESCRIPTION
**Pull request by pradeepananth**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

This is a clean up PR for removing deprecated AFR endpoints from Live Share Host file.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1. This is a clean up PR for removing deprecated AFR endpoints:
`ordererEndpoint`
`storageEndpoint`
from Live Share Host file.

### Validation performed:

1. Removed deprecated endpoints in Live Share SDK  https://github.com/microsoft/live-share-sdk/blob/046a85e2fe778f1e84559f6c90992ecc17e6d5bc/packages/live-share/src/LiveShareClient.ts#L138 
2. YouTube app 

### Unit Tests added:

> Unit tests are required for all changes. If no unit tests were added as part of this change, please explain why they aren't necessary.

No, as they already exist and updated with the new endpoint.

### End-to-end tests added:

No

### Change file added:

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

Yes

### CODE REVIEW

**Review by Stevenic — APPROVED** [review]

**Comment by erinha** [comment]
Looks like you're missing a changefile. Please see contributing.md for guidelines for how to create one. Since this interface is marked with the `@beta` tag, I think the change type is probably patch.

**Review by erinha — CHANGES_REQUESTED** [review]
Please add changefile

File: change/@microsoft-teams-js-77061c12-8716-4a1d-8a14-c09b9d1041e4.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Remove legacy AFR endpoints",
```
Comment (erinha) [inline change/@microsoft-teams-js-77061c12-8716-4a1d-8a14-c09b9d1041e4.json:3]: Change file guidelines require past tense, i.e., "Removed legacy endpoints from `IFluidTenantInfo` interface." (I don't know if any teams-js developers would know what "AFR" stands for, but if so, feel free to add that before "endpoints" in the message.)

Comment (erinha) [reply change/@microsoft-teams-js-77061c12-8716-4a1d-8a14-c09b9d1041e4.json:3]: Please note use of backticks as well

**Review by erinha — CHANGES_REQUESTED** [review]
Changefile formatting.

**Review by erinha — APPROVED** [review]

**Review by Stevenic — APPROVED** [review]

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**